_examples/golang-sse/webapp/src/rpc.gen.ts

```diff
@@ -478,8 +485,13 @@ export interface WebrpcStreamOptions<T> extends WebrpcOptions {
   onOpen?: () => void;
   onClose?: () => void;
 }
+
 export interface WebrpcOptions {
   headers?: HeadersInit;
-  signal?: AbortSignal;
+  signal?: AbortController;
```

Comment:
This is confusing; the rest of the methods (the non-SSE ones) takes the AbortSignal not the Controller.

It's also how native fetch works [signal](https://developer.mozilla.org/en-US/docs/Web/API/RequestInit#signal).

I can not think of a pretty solution for this; should we rename the field to controller?

Reply:
Yes, I think `AbortSignal` is all we need .. as an input arg, and also as a return arg (aka `abort()` function)